packages/pilot/src/containers/RecipientTable/index.js

```diff
@@ -110,9 +110,10 @@ const RecipientTable = ({
                     </div>
                     <Pagination
                       currentPage={pagination.offset}
-                      totalPages={pagination.total}
-                      onPageChange={onPageChange}
                       disabled={loading}
+                      format="single"
+                      onPageChange={onPageChange}
+                      totalPages={pagination.total}
```

Comment:
Acho melhor fazer um destructuring na linha 39 desse arquivo e utilizar `total` diretamente.
```
  pagination: {
    total,
    offset
  },
```

```suggestion
                      totalPages={total}
```

Reply:
Feito! :smile: